xml/admin_gui_dashboard.xml

```diff
@@ -953,9 +954,108 @@
     </para>
     <para>
      Click the down arrow in the top left of the table heading next to the
-     <guimenu>Edit</guimenu> button and select the task you want to perform.
+     <guimenu>Create</guimenu> button and select the task you want to perform.
     </para>
    </tip>
+   <sect3 xml:id="dashboard-cluster-osds-add">
+    <title>Adding OSDs</title>
+    <para>
+     To add new OSDs, follow these steps:
+    </para>
+    <procedure>
+     <step>
+      <para>
+       Verify that there are storage devices <emphasis>available</emphasis> on
+       cluster nodes (to see when a device is <emphasis>available</emphasis>,
+       see <xref linkend="deploy-cephadm-day2-service-osd"/>). Then click the
```

Comment:
This seems a little backwards to me.

I propose, rather than "verify that there are devices AVAILABLE on cluster nodes", we flip it to something like:

"Verify that some cluster nodes have storage devices whose status is AVAILABLE." 

Thoughts?